code/datums/abnormality/datum/abnormality.dm

```diff
@@ -65,6 +70,8 @@
 	current.toggle_ai(AI_OFF)
 	current.status_flags |= GODMODE
 	current.setDir(EAST)
+	if (fully_understood)
```

Comment:
You could just check understanding against max_understanding for the same effect.

Reply:
Am doing so now